test/module/shared_model/bindings/BuilderTest.java

```diff
@@ -0,0 +1,145 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigInteger;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class BuilderTest {
+    static {
+        try {
+            System.loadLibrary("iroha");
+        } catch (UnsatisfiedLinkError e) {
+            System.err.println("Native code library failed to load. \n" + e);
+            System.exit(1);
+        }
+    }
+
+    private Keypair keys;
+    private ModelBuilder builder;
+
+    ModelBuilder base() {
+        return new ModelBuilder().txCounter(BigInteger.valueOf(123))
+                .createdTime(BigInteger.valueOf(System.currentTimeMillis()))
```

Comment:
Maybe we need to think about the possibility of setting counter automatically from the native library.